microsoft-edge/devtools-guide-chromium/whats-new/2021/07/devtools.md

```diff
@@ -0,0 +1,288 @@
+
+1. In the **Styles** pane of the **Elements** tool, right-click a declaration in a style rule.
+1. Select **Copy declaration as JS** or **Copy all declarations as JS**.
+1. Paste the copied CSS into a JavaScript file in your text editor, such as Visual Studio Code.  For example: `'--more-link': 'lime'`.
+
+:::image type="complex" source="../../media/2021/07/copy-declaration-as-js.msft.png" alt-text="Context menu for a style rule, including 'Copy declaration as JS' and 'Copy all declarations as JS' commands" lightbox="../../media/2021/07/copy-declaration-as-js.msft.png":::
+   Context menu for a style rule, including **Copy declaration as JS** and **Copy all declarations as JS** commands
+:::image-end:::
+
+To learn more about viewing and changing CSS, navigate to [CSS reference](../../../css/reference.md).
+
+<!-- no crbug -->
+
+
+<!-- 7 -->
+## Easier customization of User-Agent Client Hints
+
+<!-- Title: Send as many (or as few) Client Hints as you want -->
+<!-- Subtitle: Updated UI for User-Agent Client Hints in Emulated Devices settings and in the Network conditions tool. -->
+
+User-Agent Client Hints makes browser information more accessible than a semicolon-delimited User-Agent string, and improves site compatibility.  Initially, User-Agent Client Hints were time-consuming to test and debug.  There was less control over the client hints, and client hints had to be filled-in properly for the form to work.
+
+In this release, we redesigned the debugging experience so you can easily modify User-Agent Client Hints through a UI with multiple separate fields and controls.  Also, you can now test your custom User-Agent Client Hints and a User-Agent string simultaneously.  You can now define User-Agent Client Hints for a custom device by using DevTools **Settings** or by using the **Network conditions** tool.
+
+To define User-Agent Client Hints for a custom device by using DevTools **Settings**:
+1. Navigate to **DevTools** > **Settings** \(![Settings icon](../../../media/settings-icon-dark.msft.png)\) > **Devices**.
+1. In the **Emulated Devices** settings page, select the **Add custom device** button.
+1. Expand the **User agent client hints** section.
+1. Fill-in some of the fields.
+
+:::image type="complex" source="../../media/2021/07/ua-client-hints-in-settings.msft.png" alt-text="Defining User-Agent Client Hints for a custom device in Settings" lightbox="../../media/2021/07/ua-client-hints-in-settings.msft.png":::
+   Defining User-Agent Client Hints for a custom device in **Settings**
+:::image-end:::
+
+To define User-Agent Client Hints for a custom device by using the **Network conditions** tool:
+1. In **DevTools**, select the **Network conditions** tool.  You might need to select **More Tools** (+) and then select **Network conditions**.
```

Comment:
@erdraud, I think this is another one where we could move these steps into a new doc or update ones like this: https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/device-mode/override-user-agent and that would let us keep the announcement a bit shorter. We could just have the 2 images in a 2 column format showing the new UI in Settings and Network conditions.

If you agree with this plan, please put up a PR that either adds a new doc or updates an existing one for this feature

Reply:
I removed the steps.  Probably no other action is needed, because an equivalent PR has been merged already; the updated articles already have the equivalent writeup of the steps:
o	https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/device-mode/#set-user-agent-client-hints 
o	https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/network/reference#set-user-agent-client-hints
Step 1 means right-click _in the rendered webpage_ and then select Inspect.
o	Related: https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/device-mode/override-user-agent  - Override the user agent string from Microsoft Edge DevTools